### PR TITLE
LPD-31856 Fix activedescendant

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:echo-playwright

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
Failures are not related

File: modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts [hunk 1]
```diff
@@ -105,3 +105,50 @@ test('Shows correct options in picklist field selected as title in related objec
 	await expect(page.getByText('Plastic', {exact: true})).toBeVisible();
 	await expect(page.getByText('Carton', {exact: true})).toBeVisible();
 });
+
+test(
+	'Checks that an info message appears when categorization is disabled',
```
Comment (veroglez) [inline modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts:110]: I think the name of the test is the result of a copy/paste :smiling_face_with_tear:

**Comment by veroglez** [comment]
@victorg1991 the code looks perfect! I only left a comment to change the name of the test

File: modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts (hunk 1 above)
Comment (victorg1991) [reply modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts:110]: Ups

Comment (victorg1991) [reply modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts]: fixed :D

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward:force

**Pull request closed without merging**